independent-publisher-connectors/Notion/apiDefinition.swagger.json

```diff
@@ -0,0 +1,677 @@
+                  "description": "A cursor returned used to request the next page of results. Treat this as an opaque value",
+                  "title": "Next cursor"
+                },
+                "has_more": {
+                  "type": "boolean",
+                  "description": "Whether or not the block has more blocks or not",
+                  "title": "Has more",
+                  "enum": [
+                    "",
+                    "true",
+                    "false"
+                  ]
+                }
+              }
+            }
+          }
+        },
+        "summary": "Retrieve block children",
+        "description": "Returns a paginated array of child block objects contained in the block using the ID specified",
+        "operationId": "Retrieve_block_children",
+        "parameters": [
+          {
+            "name": "block_id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Block id",
+            "description": "Identifier for a block"
+          },
+          {
+            "name": "page_size",
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "default": 100,
+            "x-ms-summary": "Page size",
+            "description": "The number of items from the full list desired in the response. Maximum: 100"
+          },
+          {
+            "name": "Notion-Version",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "2021-08-16",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/databases/{database_id}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object"
+            }
+          }
+        },
+        "summary": "Retrieve a database",
+        "operationId": "Retrieve_a_database",
+        "description": "Retrieves a Database object using the ID specified.",
+        "parameters": [
+          {
+            "name": "database_id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Identifier for a Notion database",
+            "x-ms-summary": "Database id"
+          },
+          {
+            "name": "Notion-Version",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "2021-08-16",
+            "x-ms-visibility": "internal"
+          }
+        ]
+      }
+    },
+    "/search": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "object": {
+                  "type": "string",
+                  "description": "object",
+                  "title": "",
```

Comment:
@sekharmalla Please remove **all** the empty operations until unless you have a specific reason to keep them.